packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache

```diff
@@ -1,8 +1,10 @@
 <div class="card">
-  <header class="hidden" id="fxa-connect-another-device-header"><h1 class="card-header mb-8">{{#t}}Connect another device{{/t}}</h1></header>
+  <header id="fxa-connect-another-device-header" class="{{^isFirefoxDesktop}}hidden{{/isFirefoxDesktop}}"><h1 class="mb-8 text-left text-base">{{#t}}Connect another device{{/t}}</h1></header>
```

Comment:
Will `text-left` work w/ RTL languages, or should this be [`text-start`](https://tailwindcss.com/docs/text-align)?

Reply:
It should be `text-start`, thanks. I was planning to circle back on this to create a new class based on `.card` that aligns left/right instead of on individual elements since we can use it in other parts of the flow.